### PR TITLE
Add wrapping `<p/>` around rich text fields populated with plain text

### PR DESCRIPTION
**Pull request by lennym**
Some description fields on research outputs in Squidex return content as plain text strings rather than rich text html. This causes them to migrate as empty because the parser removes top-level `text` nodes.

When a rich text field contains no html tags then wrap the text with a single `<p/>` before parsing so that the top level node is a `paragraph` as expected.

### CODE REVIEW

File: packages/cms-data-sync/src/utils/rich-text.ts [hunk 1]
```diff
@@ -111,6 +119,7 @@ export const convertHtmlToContentfulFormat = (html: string) => {
   processedHtml = removeStylingTagsWrappingIFrameTags(processedHtml);
   processedHtml = removeStylingTagsWrappingImgTags(processedHtml);
   processedHtml = wrapIframeWithPTag(processedHtml);
+  processedHtml = wrapPlainTextWithPTag(processedHtml);
 
   // processedHtml = addPTagsToIframeTagNotWrappedByAnything(processedHtml);
```
Comment (gabiayako) [inline packages/cms-data-sync/src/utils/rich-text.ts:124]: I think I've left this comment by mistake 🙈 could you erase before merging?

**Review by gabiayako — APPROVED** [review]
Nice 🎉

File: packages/cms-data-sync/src/utils/rich-text.ts (hunk 1 above)
Comment (lennym) [reply packages/cms-data-sync/src/utils/rich-text.ts:124]: Do you mean uncomment, or remove the line completely?

Comment (gabiayako) [reply packages/cms-data-sync/src/utils/rich-text.ts:124]: remove it completely :)

Comment (lennym) [inline packages/cms-data-sync/src/utils/rich-text.ts:124]:
```suggestion
```

**Review by gabiayako — APPROVED** [review]

**Pull request merged**